packages/manager/apps/hub/src/routing.js

```diff
@@ -50,11 +50,8 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
           ),
       numberOfServices: /* @ngInject */ ($http) =>
         $http
-          .get('/services', {
-            headers: {
-              'X-Pagination-Mode': 'CachedObjectList-Pages',
-              'X-Pagination-Size': 5,
-            },
+          .get('/hub/services', {
+            serviceType: 'aapi',
```

Comment:
**nitpick:** This change can be costly because `hub` 2API fetches the entire list of expanded services and this can take more time when the user has a huge set of services. Moreover, we dont use the data, so we are over fetching. Instead still continue to use `/services` as shown below.
```suggestion
          .get('/services', {
                headers: {
                  'X-Pagination-Size': 5,
                },
            })
            .then((data) => return data.data.length)
            .catch(() => return null);
```